libs/datamodel/connectors/sql-datamodel-connector/src/postgres_datamodel_connector.rs

```diff
@@ -140,6 +141,24 @@ impl Connector for PostgresDatamodelConnector {
                     _ => {}
                 }
             }
+            if native_type_name == SMALL_SERIAL_TYPE_NAME
+                || native_type_name == SERIAL_TYPE_NAME
+                || native_type_name == BIG_SERIAL_TYPE_NAME
+            {
+                if let Some(default_value) = field.default_value() {
+                    match default_value {
+                        DefaultValue::Single(_) => {
```

Comment:
nitpick: you can collapse the match to reduce the nesting here, something like:

`if let Some(DefaultValue::Single(_)) = field.default_value() { return Err(...) }`